draft-protected-headers.in.md

```diff
@@ -100,7 +100,61 @@ For the purposes of this document, we define the following concepts:
    * *Protected Headers* are any headers protected by the scheme described in this document.
    * *Exposed Headers* are any headers outside the Cryptographic Payload (protected or not).
    * *Obscured Headers* are any Protected Headers which have been modified or removed from the set of Exposed Headers.
-   * *Legacy Display Part* is a MIME construct which guarantees visibility of data from the Original Headers which may have been removed or obscured from the Unprotected Headers.
+   * *Legacy Display Part* is a MIME construct which provides visibility for users of legacy clients of data from the Original Headers which may have been removed or obscured from the Exposed Headers. It is defined in {{legacy-display}}.
+   * *User-Facing Headers* are explained and enumerated in {{user-facing-headers}}.
+   * *Structural Headers* are documented in {{structural-headers}}.
+
+
+### User-Facing Headers {#user-facing-headers}
+
+Of all the headers that an e-mail message may contain, only a handful are typically presented directly to the user.
+The user-facing headers are:
+
+ - `Subject`
+ - `From`
+ - `To`
+ - `Cc`
+ - `Date`
+
+The above is a complete list.  No other headers are considered "user-facing".
```

Comment:
Should all defined terms appear in ```` or with 1st letter capitalized?,
s/"user-facing"/`User-Facing`/

Reply:
I've been trying to use `code` style to indicate literal textual elements, and Upper Case to indicate defined terms.